docs/releases/1.9.txt

```diff
@@ -360,6 +360,15 @@ Miscellaneous
 * ``CommaSeparatedIntegerField`` validation has been refined to forbid values
   like ``','``, ``',1'``, and ``'1,,2'``.
 
+* ``ModelState`` has been changed to check that the ``to`` and ``through``
```

Comment:
Point taken. I thought I had already seen packages out there overriding the `state_forwards()` method of operations, although it's private API. But so far the Github search doesn't give me anything: https://github.com/search?utf8=%E2%9C%93&q=django+migrations+Operation+state_forwards&type=Code&ref=searchresults